include/ebpf_program_types.h

```diff
@@ -43,6 +43,8 @@ typedef struct _ebpf_program_info
     ebpf_program_type_descriptor_t program_type_descriptor;
     uint32_t count_of_helpers;
     MIDL([size_is(count_of_helpers)]) ebpf_helper_function_prototype_t* helper_prototype;
```

Comment:
```suggestion
    uint32_t count_of_program_type_specific_helpers;
    MIDL([size_is(count_of_helpers)]) ebpf_helper_function_prototype_t* program_type_specific_helper_prototype;
```

Reply:
Changed to `count_of_program_specific_helpers` and `program_specific_helper_prototype`